platform/darwin/src/MGLThreadPool_Private.h

```diff
@@ -0,0 +1,18 @@
+#import <Foundation/Foundation.h>
+
+#include <mbgl/util/default_thread_pool.hpp>
+
+@interface MGLThreadPool : NSObject
+@end
+
+@interface MGLThreadPool (Private)
```

Comment:
No need for a class extension, since the whole header is already internal to the project. A class extension would need to go in a separate header from the public header anyways.